bugbug/repository.py

```diff
@@ -19,7 +19,7 @@
 COMMITS_DB = 'data/commits.json'
 db.register(COMMITS_DB, 'https://www.dropbox.com/s/mz3afgncx0siijc/commits.json.xz?dl=1')
 
-Commit = namedtuple('Commit', ['node', 'author', 'desc', 'date', 'bug'])
+Commit = namedtuple('Commit', ['node', 'author', 'desc', 'date', 'bug', 'no_of_backouts'])
```

Comment:
I think a given commit can be backed out only once, so this should be a boolean `ever_backedout`

Reply:
yes as far as I saw the outputs it was giving only one node.And yes I'll change it.